File: projects/store-or/src/test/java/com/robinfood/storeor/mocks/dto/configurationposbystore/StorePosDTOMock.java
package com.robinfood.storeor.mocks.dto.configurationposbystore;

import com.robinfood.storeor.dtos.configurationposbystore.StorePosDTO;

import java.util.Collections;
import java.util.List;

public class StorePosDTOMock {

    public List<StorePosDTO> storePosDTOs = List.of(
            StorePosDTO.builder()
                    .id(1L)
                    .name("caja1")
                    .resolutions(Collections.emptyList())
                    .build(),
            StorePosDTO.builder()
                    .id(2L)
                    .name("caja2")
                    .resolutions(Collections.emptyList())
                    .build()
    );
}

File: projects/posv2/app/src/test/kotlin/com/robinfood/app/mocks/dailyreportevoucher/StorePosDTOMock.kt
package com.robinfood.app.mocks.dailyreportevoucher

import com.robinfood.core.dtos.posrelatedtoastore.StorePosDTO

class StorePosDTOMock {

    val storePosDTOs = listOf(
        StorePosDTO(
            1L,
            "pos 1",
            ResolutionDTOsMock().resolutionDTOs
        )
    )
}
